extensions/core/deployment/src/main/java/org/apache/camel/quarkus/core/deployment/CamelBeanBuildItem.java

```diff
@@ -18,6 +18,8 @@
 
 import java.util.Objects;
 
+import javax.annotation.Nullable;
```

Comment:
I never used jsr305 and my naive expectation is that findbugs/spotbugs is requited to enforce it. Or it works differently? 

jsr305 artifact currently being a transitive dep of guava suggests to be cautious. Can't this potentially become a maintenance issue?

Reply:
it is primary for documentation purpose here